### PR TITLE
add, list, delete a user

### PR DESCRIPTION
**Pull request by quanghoan**
I'm new to Laravel, I need your help to improve my code, any comments will be appreciated. Thank you!
I do: basic functions of adding, deleting and listing users.

### CODE REVIEW

**Comment by kakeyang** [comment]
Where do you write the code for sort and validation?

File: app/Http/routes.php [hunk 1]
```diff
@@ -1,16 +1,44 @@
...
+  return redirect('/');
+});
+
+/**
+ * Delete An Existing user
+ */
+Route::delete('/user/{id}', function ($id) {
+  User::findOrFail($id)->delete();
+
+  return redirect('/');
```
Comment (anh-pham191) [inline app/Http/routes.php]: Can  you put this into UserController and make a Route::resourece instead ?

File: resources/views/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+        <!-- Add user Button -->
+        <div class="form-group">
+          <div class="col-sm-offset-3 col-sm-6">
+            <button type="submit" class="btn btn-default">
+                <i class="fa fa-plus"></i> Add user
+            </button>
+          </div>
+        </div>
+      </form>
```
Comment (anh-pham191) [inline resources/views/index.blade.php]: You may want to try FormBuilder instead, it automatically generates csrf token

**Comment by quanghoan** [comment]
@kakeyang they're in routes.php file.

**Comment by quanghoan** [comment]
@tuananhpham94 I'll try it, thank you .

**Pull request merged**